testcharms/charm-hub/charms/juju-qa-test-v2/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 juju-qa@canonical.com
+# See LICENSE file for licensing details.
+name: juju-qa-test
```

Comment:
s/juju-qa-test/juju-qa-test-v2
Otherwise juju will pick up this as the name, and you won't be able to deploy both at the same time without renaming there

Reply:
The name should be updated in all the charm files